docs/elements/hx-panelhead/index.html

```diff
@@ -0,0 +1,11 @@
+---
+title: <hx-panelhead>
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      An <code>hx-panelhead</code> is fixed to the top of an <a href="elements/hx-panel">&lt;hx-panel&gt;</a>.
```

Comment:
Should there be an indication here that it is optional in the layout?

Reply:
@CITguy will modify in the metadata update